### PR TITLE
Add DR filtering matrices

### PR DESCRIPTION
**Pull request by krenzland**
This PR adds filter matrices for the DR output. Note that the matrices for the quad rule Dunavant is probably incorrect (compare sum of the quad weights for example).

The resample matrix is then a generalization of DR filtering. 

This PR includes the matrices used in: https://github.com/SeisSol/SeisSol/pull/978

### CODE REVIEW

**Comment by sebwolf-de** [comment]
#11 should fix the problems with the Dunavant rule.

**Comment by krenzland** [comment]
Thanks, I've included the changes. I renamed the quad rules so that the spelling agrees with the matrices already in SeisSol. The current naming includes a space in the file names, which is a bit ugly.

File: src/seissol_matrices/dr_matrices.py [hunk 1]
```diff
@@ -105,20 +105,34 @@ def resample(self):
...
-                E[i, j] = self.bf2_generator.eval_basis(p, i)
-        W = np.eye(m)
-        for i in range(m):
-            W[i, i] = weights[i]
+                E[i, j] = self.bf2_generator.eval_basis(p, i) * weights[j]
 
         mass = self.dg2_generator.mass_matrix()
 
-        return np.dot(E.T, np.linalg.solve(mass, np.dot(E, W)))
+        V = np.linalg.inv(mass) @ E
```
Comment (sebwolf-de) [inline src/seissol_matrices/dr_matrices.py]: Why the inverse, IMHO `solve` is better, because `inv` is less stable.

**Review by sebwolf-de — CHANGES_REQUESTED** [review]

File: src/seissol_matrices/dr_matrices.py (hunk 1 above)
Comment (krenzland) [reply src/seissol_matrices/dr_matrices.py]: I'm pretty sure it's identical for matrices (mass is diagonal anyway!)

Comment (krenzland) [reply src/seissol_matrices/dr_matrices.py]: Actually it is slightly larger than I expected for very large orders :O

**Review by sebwolf-de — APPROVED** [review]

**Pull request merged**